app/model/report/ApplicationForOnlineTestPassMarkReportItem.scala

```diff
@@ -37,7 +37,9 @@ object ApplicationForOnlineTestPassMarkReportItem {
   implicit val applicationForOnlineTestReportItemFormat = Json.format[ApplicationForOnlineTestPassMarkReportItem]
 
   // If you add a custom apply() to a case class companion object then Json.reads and Json.writes fail
-  def create(a: ApplicationForOnlineTestPassMarkReport, fr: Option[AssessmentScoresAllExercises]): ApplicationForOnlineTestPassMarkReportItem = {
+  def create(a: ApplicationForOnlineTestPassMarkReport,
+             fr: Option[AssessmentScoresAllExercises],
```

Comment:
Also readableVariableNames

Reply:
Already fixed in a PR coming off the back of this